packages/ui/src/components/SliderSteps/SliderStep.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Box } from '@chakra-ui/react';
+
+export interface SliderStepProps {
+  current: boolean;
+}
+
+export const SliderStep = ({ current }: SliderStepProps) => {
+  const bgColor = current ? 'brand.500' : 'gray.300';
+  const horizontalMargin = current ? '2px' : '10px';
+
+  return (
+    <Box
+      data-testid={current ? 'currentSliderStep' : 'baseSliderStep'}
```

Comment:
💭 We can make the entire component more accessible and as a bonus, there would be no need for `data-testid`.
1. Add `role="tablist"` to the SliderSteps component.
2. Add `role="tab" tabindex="0"` here.
3. Add `aria-selected={current}` here.